File: problems/music-scales/solution/MusicScales.java
import java.util.Scanner;

public class MusicScales {
    public static String getNoteInScale(int position) {
        String[] cMajorScale = {"C", "D", "E", "F", "G", "A", "B", "C"};
        // Adjust for zero-based indexing
        return cMajorScale[position - 1];
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Parse the input
        int position = scanner.nextInt();

        String note = getNoteInScale(position);
        System.out.println(note);
        

        scanner.close();
    }
}

File: problems/B-music-scales/solution/music-scales.py
def get_note_in_scale(position):
    c_major_scale = ["C", "D", "E", "F", "G", "A", "B", "C"]
    # Adjust for zero-based indexing
    return c_major_scale[position - 1]

# Parse the input
position = int(input())

# Validate input
if 1 <= position <= 8:
    # Call the function and print the result
    note = get_note_in_scale(position)
    print(note)
